examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/checkbox/_cq_template.xml

```diff
@@ -3,4 +3,6 @@
   jcr:primaryType="nt:unstructured"
   jcr:title="Check Box"
   fieldType="checkbox"
-  enum="[true,false]"/>
+  checkedValue="true"
```

Comment:
Shouldn't this be boolean ?

Reply:
was saving this as string, Will change it to "on" and "off" to align with headless examples and to avoid confusion